src/StripeEmitter.ts

```diff
@@ -0,0 +1,51 @@
+/** @private
+ * (For internal use in stripe-node.)
+ * Wrapper around the Event Web API. */
+class _StripeEvent extends Event {
+  data: any;
+  constructor(eventName: string, data: any) {
+    super(eventName);
+    this.data = data;
+  }
+}
+
+/** Minimal EventEmitter wrapper around EventTarget. */
+class StripeEmitter {
```

Comment:
Let's write some tests that run both on `StripeEmitter` and `EventEmiter`. We need to confirm that `StripeEmitter` and has the same behavior as `EventEmiter`.

Reply:
This is done in [`PlatformFunctions.spec.ts`](https://github.com/stripe/stripe-node/pull/1668/files#diff-f0d0bd6c151c9a6aff19b6a75d08af3bb6c01c410b4494b9dd1e7144481d0a45L14-L18)